frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -729,10 +729,18 @@ const ManageHostsPage = ({
 
       let sort = sortBy;
       if (sortHeader) {
+        let direction = sortDirection
+        if (sortHeader === 'last_restarted') {
+          if (sortDirection === 'asc') {
+            direction = 'desc'
+          } else {
+            direction = 'asc'
+          }
+        }
```

Comment:
Sorry don't have a lot of context on this. Why the need to send the opposite sort direction only for last_restarted? It seems strange for this column to have different sort behaviour than all the others.

Reply:
Was discussed up above: https://github.com/fleetdm/fleet/pull/14878#discussion_r1380221391